content/source/docs/enterprise/install/active-active.html.md

```diff
@@ -230,9 +230,9 @@ For example:
 
 
 
-##### Add Common Configuration
+##### Add Common Encryption Password
```

Comment:
Since the encryption password is now the only token covered in this section, I think removing "Common" will help clarify the topic.

```suggestion
##### Add Encryption Password
```